nxdrive/gui/dialog.py

```diff
@@ -1,44 +1,40 @@
 # coding: utf-8
```

Comment:
This file has nothing to do with a "dialog" anymore, maybe we can change it to `api.py`?